scripts/ensure_min_python_is_tested.py

```diff
@@ -0,0 +1,60 @@
+# run via `tox r -e check-min-python-is-tested`
+import pathlib
+import subprocess
+import sys
+
+import ruamel.yaml
+
+YAML = ruamel.yaml.YAML(typ="safe")
+REPO_ROOT = pathlib.Path(__file__).parent.parent
+
+proc = subprocess.run(
+    ["python", "-m", "mddj", "read", "requires-python", "--lower-bound"],
+    check=True,
+    capture_output=True,
+    cwd=REPO_ROOT,
+)
+requires_python_version = proc.stdout.decode().strip()
```

Comment:
This looks like an ideal place for `subprocess.check_output()`, which gets us a decoded STDOUT. I think the code below is equivalent.

```suggestion
requires_python_version = subprocess.check_output(
    ["python", "-m", "mddj", "read", "requires-python", "--lower-bound"],
    cwd=REPO_ROOT,
    text=True,
).strip()
```